src/redux/apis/baseApi.js

```diff
@@ -3,8 +3,7 @@ import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
 export const baseApi = createApi({
   reducerPath: 'baseApi',
   baseQuery: fetchBaseQuery({
-
-    baseUrl: 'https://slimmomproject.herokuapp.com/api',
```

Comment:
Max, I suggested to test locally, before backend pull request is accepted by you :)
Alexa and I will modify to normal backend in the next pull.